content/en/docs/reference/access-authn-authz/kubelet-authn-authz.md

```diff
@@ -115,8 +115,11 @@ flags passed to the API server is authorized for the following attributes:
 * verb=\*, resource=nodes, subresource=proxy
 * verb=\*, resource=nodes, subresource=stats
 * verb=\*, resource=nodes, subresource=log
-* verb=\*, resource=nodes, subresource=spec
```

Comment:
also delete `/spec` from the table above?

Reply:
OK.